pkg/client/interface.go

```diff
@@ -274,4 +274,18 @@ type SessionQueryClient interface {
 		serviceId string,
 		blockHeight int64,
 	) (*sessiontypes.Session, error)
+	// GetParams queries the chain for the current session module parameters.
+	GetParams(ctx context.Context) (*sessiontypes.Params, error)
+	// GetSessionGracePeriodEndHeight returns the height at which the grace period for
+	// the session ending with sessionEndHeight elapses.
+	GetSessionGracePeriodEndHeight(ctx context.Context, sessionEndHeight int64) (int64, error)
+	// GetSessionGracePeriodBlockCount returns the number of blocks in the grace period
+	// for the session which includes queryHeight.
+	GetSessionGracePeriodBlockCount(ctx context.Context, queryHeight int64) (uint64, error)
+	// IsWithinGracePeriod returns true if the grace period for the session ending with
+	// sessionEndHeight has not yet elapsed, given currentHeight.
+	IsWithinGracePeriod(ctx context.Context, sessionEndHeight, currentHeight int64) (bool, error)
+	// IsPastGracePeriod returns true if the grace period for the session ending with
+	// sessionEndHeight has elapsed, given currentHeight.
+	IsPastGracePeriod(ctx context.Context, sessionEndHeight, currentHeight int64) (bool, error)
```

Comment:
Given that we have `IsWithinGracePeriod`, why do we need `IsPastGracePeriod` as well?

Reply:
Probably not. Looking at the usages, it seems like it would be better to keep `IsPastGracePeriod` (maybe rename to `IsGracePeriodElapsed`) and get rid of `IsWithinGracePeriod`.